src/applications/disability-benefits/526EZ/components/PaymentView.jsx

```diff
@@ -33,23 +30,66 @@ const PaymentView = response => {
     );
   };
 
-  if (accountType !== NOBANK) {
-    accountNumberString = <p>Account number: {mask(accountNumber, 4)}</p>;
-    routingNumberString = <p>Bank routing number: {mask(routingNumber, 4)}</p>;
-    bankNameString = (
-      <p>Bank name: {bankName || srSubstitute('', 'is blank')}</p>
+  if (!accountType || !accountNumber || !routingNumber) {
+    // should we also check bankName?
+    return (
+      <div>
+        <PaymentDescription />
+        <div className="blue-bar-block">
+          <p>
+            <strong>
+              We don’t have any bank account information for you in our system.
+            </strong>
+            <br />
+            You can still complete this form, but we’ll need your payment
+            information in order to pay you compensation.
+          </p>
+          <p>
+            <strong>You can update your payment information a few ways:</strong>
+          </p>
+          <ul>
+            <li>
+              Fill out and send us a Direct Deposit Enrollment form (VA Form
+              24-0296).{' '}
+              <a
+                href="https://www.vba.va.gov/pubs/forms/vba-24-0296-are.pdf"
+                target="_blank"
+              >
+                Download VA Form 24-0296
+              </a>
+              , <strong>or</strong>
```

Comment:
Eh, Prettier. I'm developing a love / hate relationship with this.

Reply:
Yeah. To be fair, at least I didn't have to think about it - I just wrote one big long line and got it all fixed up on save.